docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -159,7 +159,175 @@ the pre-aggregations are rebuilt.
 These refreshes can be done on-demand, or [in the background as a scheduled
 process][ref-caching-preaggs-bk-refresh].
 
+## Ensuring pre-aggregations are targeted by queries
+
+### Additivity
```

Comment:
Maybe it's good to have a one-liner here: "If your measure has one of these types ..., it's addi(c)tive. Here's what it means."

Reply:
Same for "leafs".

Reply:
I was sort of aiming to keep Cube.js terminology out of the "Additivity" sections for clarity's sake. My plan was to tie it together at the end with a tl;dr-ish explanation.